docs/modeling/example-fitting-model-sets.rst

```diff
@@ -1,37 +1,120 @@
 Fitting Model Sets
 ==================
 
-Fitting a polynomial model to multiple data sets simultaneously::
+Astropy model sets let you fit the same (linear) model to lots of independent
+data sets. It does the looping for you, and can be significantly faster than
```

Comment:
Can you provide some estimate of the savings?  Maybe just a reasonable example with rough timings?  After reading this, I wondered if this was a factor of 10% improvement, factor of 2, factor of 10, ...

Reply:
I would not use the word `looping`. It actually solves a set of linear equations simultaneously to find the exact solution. This is the reason also why it doesn't work with non-linear models which require an iterative fitting and looping.

Reply:
Added a time estimate. Factor of more than 600 for the use case of a 10x500x500 cube on my MacBook.

Reply:
Revised to say that it solves the linear equations simultaneously.